services/app-web/src/pages/StateSubmission/ContractDetails/ContractDetails.tsx

```diff
@@ -466,14 +466,8 @@ export const ContractDetails = ({
     return (
         <Formik
             initialValues={contractDetailsInitialValues}
-            onSubmit={(values, { setSubmitting }) => {
-                return handleFormSubmit(values, setSubmitting, {
-                    shouldValidateDocuments: true,
-                    redirectPath:
-                        draftSubmission.submissionType === 'CONTRACT_ONLY'
-                            ? `../contacts`
-                            : `../rate-details`,
-                })
+            onSubmit={() => {
```

Comment:
what happens if you leave this function out?

Reply:
formik type error

Reply:
Its a react-uswds props types error. We would need to update that external dep (which we aren't even up to date on to fix).

Honestly I would get around this by making a `MCRForm` that's just our internal form for use in this app, copy react-uswds form code which is very simple and just adjust the required props.